components/shared/QuickLink.tsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import Image from 'next/image';
+
+const QuickLink = ({icon, href, name}: {icon: string; href: string; name: string}) => {
```

Comment:
Since the Quick Links shared a similar pattern, I extracted it into one component

Reply:
Perfect